catalyst-gateway/bin/src/service/common/types/headers/retry_after.rs

```diff
@@ -36,6 +36,16 @@ pub(crate) enum RetryAfterOption {
     Some(RetryAfterHeader),
 }
 
+impl From<RetryAfterOption> for Option<RetryAfterHeader> {
```

Comment:
Revert.
The backend only uses this for documentation,  rate limiting is handled solely by the ingress.
Also, modifying the retry logic is out of scope for the ticket.